### ISSUE TITLE
Creation of New User or resetting Password is taking forever

### ISSUE THREAD

**Issue by KenjiOrigin** [nilsteampassnet/TeamPass#2970]
### Steps to reproduce
1. Freshly installed Teampass 3.0.0.8 on Debian
2. Created a few User for tests - started filling the Folder with Data
3. User forgot Password, tried to reset

### Expected behaviour
As it should work in the beginning. If I create a new user or want to change a password it should generate a new password after the "Encrypting keys - Items [0 - 200] ... Please wait!" line.

### Actual behaviour
I am logged in for over 30 Minutes now - it is still saying "Encrypting keys - Items [0 - 200] ... Please wait!".

### Server configuration
**Operating system**:
Debian 11
**Web server:**
Apache 2.4.48 
**Database:**
MariaDB 10.5.11-MariaDB-1 Debian 11
**PHP version:**
7.4.21 
**Teampass version:** 
3.0.0.8
**Teampass configuration file:**

Sorry i am not sure what you actually mean with that :(

**Updated from an older Teampass or fresh install:**
Fresh install.

### Client configuration
**Browser:**
Opera GX
(trying it with firefox right now!)
**Operating system:**
Windows 10

I am not sure which logs you need - if you need one, please tell me. I will give them to you :)

(I am not sure, maybe it takes that long because he need to re-generate the SK in every Data?)

**Comment by jdejean86** [comment]
Hello,

I have the same issue, do you find the solution ?

Regards

**Comment by Anth0nyME** [comment]
Have this problem too

**Comment by jzan** [comment]
I was going to post that I have the same problem as well, didn't happen on clean install 3.0.0.8 but now I have more than 500 passwords in 124 folders I am seeing the same behavior.  I noticed that the 500 error comes back at exactly 60 seconds in the browser console. I was able to get around this by increasing the php.ini max_execution_time = 60 from the instructions to 600 and it completes successfully now, 0-200,200-400, 400-600 then email.
